src/lsp/server/utils/serverWorkspace.ts

```diff
@@ -5,35 +5,20 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import { SFDX_PROJECT_FILE } from './constants';
-import * as fs from 'fs';
-import * as path from 'path';
 import { WorkspaceFolder } from 'vscode-languageserver';
 
-export class WorkspaceUtils {
-    // WorkspaceFolders is null if Workspace is not configured
+export class ServerWorkspace {
```

Comment:
Much of `ServerWorkspace` came from `WorkspaceUtils` which most of it was a code from previous iteration prior to LSP. The reason LSP needs a `ServerWorkspace` was because it wanted to stash `workspaceFolders` property(next line in the code) that gets passed to when the LSP server is initialized.

Rather than duplicating `WorkspaceUtils`, I created a property bag class with managing `workspaceFolders` property. Static methods are essentially the same as they were implemented when LSP feature was done.

Reply:
Hmm.... I'm not sure how I feel about `ServerWorkspace` as the name. This file was containing helper code around the concept of Workspace in an extension, and to that end `Server` or `Client` workspace doesn't make sense (not to me at least).... but I'll differ to others on this one.

Reply:
Discussed offline and will keep as is in this PR

Reply:
WorkSpaceFolder in the LSP server and VS Code are two distinct classes. In the previous workspace structure, the server and client were independent packages, so using workspace as a name made sense within a self-contained project. Now that we've consolidated everything into a single package, renaming it to ServerWorkSpace is more appropriate to distinguish it from VS Code's workspace.